housing_cooperative_base/models/lease.py

```diff
@@ -84,16 +96,14 @@ def _compute_name(self):
             lease.name = "%s/%s" % (tenant, date)
 
     @api.multi
-    @api.depends("housing_id", "room_ids", "cellar_ids")
+    @api.depends("lease_line_ids")
     def _compute_suggested_rent(self):
         for lease in self:
-            rent = lease.housing_rent or 0
-            rent += sum(lease.room_ids.mapped("rent"))
-            rent += sum(lease.cellar_ids.mapped("rent"))
-            charges = lease.housing_charges or 0
-            charges += sum(lease.room_ids.mapped("charges"))
-            charges += sum(lease.cellar_ids.mapped("charges"))
-
+            rent = 0
```

Comment:
I'm a _loop chaser_, a true pythonista writes:
```python
rent = sum(line.rent for line in lease.lease_line_ids)
charges = sum(line.charge for line in lease.lease_line_ids)
```

btw, if you're looping on lease lines, don't write `for premise in lease_line_ids`. It's not the same symbol in a human brain even if rent is on the lease line.

Reply:
Super clean, glad to learn this! Joining pythonista club...